src/Methods/Address/GetUnspents.ts

```diff
@@ -87,18 +89,18 @@ export default method({
 
       const outpoint = `${output.vout.txid}:${output.vout.n}`;
 
-      const [inscriptions, ordinals, runeBalances] = await Promise.all([
+      const [inscriptions, ordinals, hasRunes] = await Promise.all([
         db.inscriptions.getInscriptionsByOutpoint(outpoint),
         ord.getOrdinals(outpoint),
-        ord.getRuneOutputsBalancesByOutpoint(outpoint),
+        outputHasRunes(outpoint),
       ]);
 
       // DEPRECATED, REMOVE WHEN CLIENT CATCHES UP
       utxo.ordinals = ordinals;
       utxo.inscriptions = inscriptions;
-      utxo.runes = runeBalances;
+      utxo.runes = hasRunes;
```

Comment:
I think this was meant to return the rune balances on the UTXO. Can we fix this on PR68?

Reply:
Since it's not being used for now I'll merge this PR first.

Reply:
I'll add it to PR68